nautobot/extras/tables.py

```diff
@@ -493,3 +494,36 @@ class Meta(BaseTable.Meta):
             "content_type",
             "description",
         )
+
+
+class CustomFieldTable(BaseTable):
+    pk = ToggleColumn()
+    name = tables.Column(linkify=True, verbose_name="Slug")
```

Comment:
I *think* it'd be more correct to possibly do:

```python
slug = tables.Column(linkify=True, accessor="name")
```

as I'm thinking that that will allow for table sorting/filtering by "slug" instead of by "name" (e.g. `/extras/custom-fields/?sort=slug` rather than `/extras/custom-fields/?sort=name`) -- can you try this out?